Templates/templates/Java/models_extensions/BaseEntity.java.tt

```diff
@@ -40,4 +40,26 @@ import java.util.HashMap;
 <#=TypeHelperJava.CreatePropertyDef(((OdcmClass)c).Properties, c is OdcmComplexClass)#>
 <#=TypeHelperJava.CreateRawJsonObject()#>
 <#=TypeHelperJava.UpdatePropertiesWithinSetRawObject(((OdcmClass)c).Properties, c is OdcmComplexClass)#>
+<# if(c.TypeName().Equals("UploadSession")) {#>
```

Comment:
Where was IUploadSession implementation before this generated change?

Reply:
before this generated change the UploadSession type was used directly by the provider. We didn't have an interface.